beginner_source/basics/saveloadrun_tutorial.py

```diff
@@ -52,7 +52,7 @@
 ########################
 # We can then load the model like this:
 
-model = torch.load('model.pth')
+model = torch.load('model.pth', weights_only=False),
```

Comment:
nit: I would add a comment here that as noted in https://pytorch.org/docs/main/notes/serialization.html#saving-and-loading-torch-nn-modules, saving `state_dict`s is the best practice and that we use `weights_only=False` here because this is loading a model, which is a legacy use case for `torch.save`